config/nginx.conf

```diff
@@ -9,19 +9,4 @@ server {
         uwsgi_pass_request_headers on;
         uwsgi_pass_request_body on;
     }
-
-    location @webpack {
-        proxy_pass http://watch:8078;
-    }
-
-    location ^~ /static/(.*$) {
-        try_files $uri $uri/ staticfiles/$1 staticfiles/$1/ @webpack =404;
```

Comment:
If we are going to remove the webpack named location then the `@webpack` reference can be deleted but it makes sense to continue using Nginx to serve the static assets so that it is more akin to production.